### ISSUE TITLE
[9.2.3] Fatal error relating to elected titles

### ISSUE THREAD

**Issue by thetestgame** [StrangeLoopGames/EcoIssues#20329]
This error seems to occur every 3 to 4 hours and brings down the entire server when it happens.

```
System.InvalidOperationException: Nullable object must have a value.
   at Eco.Gameplay.Civics.GameValues.Values.ResidentOfDistrict.Value(IContextObject action)
   at Eco.Gameplay.Civics.Titles.ElectedTitle.CheckConditions(User user, GameValue`1 check, List`1 reasons)
   at Eco.Gameplay.Civics.Titles.ElectedTitle.CanRun(User user)
   at Eco.Gameplay.Civics.Constitutional.CivicDutyManager.BuildData(User user)
   at Eco.Gameplay.Civics.Constitutional.CivicDutyManager.Tick()
   at Eco.Gameplay.Civics.CivicsPlugin.TickAll()
   at Eco.Core.Utils.Threading.PeriodicWorkerFactory.<>c__DisplayClass0_0.<Create>b__0(CancellationToken token)
   at Eco.Core.Utils.Threading.RepeatableActionWorker.DoRepeatableWork(CancellationToken token)
   at Eco.Core.Utils.Threading.RepeatableWorkerBase.DoWork(CancellationToken token)
 
[21:24:05.022] [5] [ Info] [Eco] Saving...  
[21:24:19.590] [40] [Error] [Eco] System.InvalidOperationException: Nullable object must have a value.
   at Eco.Gameplay.Civics.GameValues.Values.ResidentOfDistrict.Value(IContextObject action)
   at Eco.Gameplay.Civics.Titles.ElectedTitle.CheckConditions(User user, GameValue`1 check, List`1 reasons)
   at Eco.Gameplay.Civics.Titles.ElectedTitle.CanRun(User user)
   at Eco.Gameplay.Civics.Constitutional.CivicDutyManager.BuildData(User user)
   at Eco.Gameplay.Civics.Constitutional.CivicDutyManager.Tick()
   at Eco.Gameplay.Civics.CivicsPlugin.TickAll()
   at Eco.Core.Utils.Threading.PeriodicWorkerFactory.<>c__DisplayClass0_0.<Create>b__0(CancellationToken token)
   at Eco.Core.Utils.Threading.RepeatableActionWorker.DoRepeatableWork(CancellationToken token)
   at Eco.Core.Utils.Threading.RepeatableWorkerBase.DoWork(CancellationToken token)
   at Eco.Core.Utils.Threading.WorkerBase.ShutdownAsync()
   at Eco.Core.Utils.Threading.WorkerExtensions.Shutdown(IWorker worker)
   at Eco.Gameplay.Civics.CivicsPlugin.Shutdown()
   at Eco.Server.PluginManager.<>c.<ShutdownAsync>b__35_0(PluginRegistration reg)
   ```
[log_210202081034.log](https://github.com/StrangeLoopGames/EcoIssues/files/5915783/log_210202081034.log)

**Comment by SlayksWood** [comment]
@thetestgame Thank you for the report. Can you provide save file to solve problem faster? If you don't want to make it publicly, you can send it to my mail kirill@strangeloopgames.com or in discord Kirill (Slayks)#6251

**Comment by Fear1essLead3r** [comment]
@SlayksWood we appear to be having a similar issue, will DM you on Discord now

**Comment by WillyWonka2197** [comment]
Here is the most recent save file.
https://drive.google.com/file/d/1-AiwOcJ3ZiOZepchnmriHZ99sPuLMXTg/view?usp=sharing

**Comment by thetestgame** [comment]
@SlayksWood Just for clarity the file Willy posted is the same one that this error keeps occurring with.

**Comment by WeaselDog** [comment]
Fixed in 9.2.4
